lib/api/include/irods/apiNumber.h

```diff
@@ -7,7 +7,7 @@
 #ifndef API_NUMBER_H__
 #define API_NUMBER_H__
 
-#define API_NUMBER(NAME, VALUE) const int NAME = VALUE;
+#define API_NUMBER(NAME, VALUE) static const int NAME = VALUE;
```

Comment:
I *think* this will go away once you rebase, but... just pinning this for now.

26fe8ada26aa169fb76b0beb91656e9f65e94577

Reply:
Oh, @korydraughn already addressed it. Carry on.

Reply:
Odd since 

> You're going to need to remove the commit for #6448 from this PR. That fix is already included in the main branch. If you don't see it, update the main branch of your fork and it should appear. After that, rebase your changes on top of main so that things compile.

If I `pull --rebase` I thought it would have exactly that effect, but I have rebased against the updated main that contains Violet's changes, etc., so everything seems to be as we intend it.

Reply:
Your PR still includes #6448. You need to do an interactive rebase and drop that commit.